script/cache_critical_dns

```diff
@@ -1,14 +1,12 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
-
+require 'bundler/setup'
```

Comment:
(Use whatever Ruby people would consider idiomatic if it contradicts my suggestion.)

If this import must be ordered before the rest, particularly `pg` and `redis`, it might help to communicate that requirement by separating it with a single blank line.

```
require 'bundler/setup'

require 'ipaddr'
require 'pg'
require 'redis'
# ...
```

(FWIW, py style is stdlib imports first, blank line, then third-party imports.  But we don't have any equivalent to `bundler/setup` and relative ordering is not practically significant, so this is a bit new to me.)